src/metabase/api/card.clj

```diff
@@ -207,6 +209,175 @@
       (when-not (Boolean/parseBoolean ignore_view)
         (events/publish-event! :card-read (assoc <> :actor_id api/*current-user-id*))))))
 
+(defn- card-columns-from-names
+  [card names]
+  (when-let [names (set names)]
+    (filter #(names (:name %)) (:result_metadata card))))
+
+(defmulti series-are-compatible?
```

Comment:
Can this be moved down closer to the method implementations?

Reply:
yep a9ef5737f0